src/bluetooth_obex.rs

```diff
@@ -0,0 +1,165 @@
+extern crate dbus;
+use self::dbus::arg::{Dict, Variant};
+use self::dbus::Path as ObjectPath;
+use self::dbus::{BusType, Connection, Message, MessageItem, Props};
+use std::collections::HashMap;
+use std::error::Error;
+use std::path::Path;
+use std::thread::sleep;
+use std::time::Duration;
+
+use bluetooth_device::BluetoothDevice;
+
+const OBEX_BUS: &str = "org.bluez.obex";
+const OBEX_PATH: &str = "/org/bluez/obex";
+const OBJECT_PUSH_INTERFACE: &str = "org.bluez.obex.ObjectPush1";
+const CLIENT_INTERFACE: &str = "org.bluez.obex.Client1";
+const TRANSFER_INTERFACE: &str = "org.bluez.obex.Transfer1";
+
+pub enum SessionTarget {
+    Ftp,
+    Map,
+    Opp,
+    Pbap,
+    Syncr,
```

Comment:
Is there any reason why you call it `Syncr`, if you later call it `"sync"`?

Reply:
Actually I named this variant `Sync`, but then I found out that it may conflict with the Rust `Sync` trait, which is present in the global namespace. Maybe you have some idea how to name the `"sync"` variant in better way? `Sync_` maybe?

Variant comes from here: https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/doc/obex-api.txt#n30